client/my-sites/domains/domain-management/list/domain-row.jsx

```diff
@@ -367,7 +379,6 @@ class DomainRow extends PureComponent {
 					disabled={ disabled || isBusy }
 					onClick={ this.stopPropagation }
 					toggleTitle={ translate( 'Options' ) }
-					icon={ <Icon icon={ moreVertical } size={ 28 } className="gridicon" /> }
```

Comment:
Looks like a lot of the changes to the Ellipsis icon look/behavior derive from this change and similar changes in other files - basically justing using the default EllipsisMenu behavior. Also looks like someone had been trying to override the default behavior of EllipsisMenu by providing an icon prop? Makes me wonder why that was being done in the first place.

Reply:
> Makes me wonder why that was being done in the first place.

I'm unsure either, but it looks like it might be standard practice throughout the domains views. I just noticed vertical ellipses in other domains pages as well